phoebe/frontend/bundle.py

```diff
@@ -10817,7 +10819,7 @@ def compute_pblums(self, compute=None, model=None, pblum=True, pblum_abs=False,
 
                     logger.info("estimating pblum for {}@{} using atm='{}' and stefan-boltzmann approximation".format(dataset, component, atm))
                     # requiv in m, Inorm in W/m**3, ldint unitless, ptfarea in m -> pblum_abs in W
-                    pblums_abs[dataset][component] = 4 * np.pi * requivs[component]**2 * abs_normal_intensities * ldint * ptfarea
+                    pblums_abs[dataset][component] = float(4 * np.pi * requivs[component]**2 * abs_normal_intensities * ldint * ptfarea)
```

Comment:
which of these are lists/arrays (I think `abs_normal_intensities` and `ldint` since those are being queried from the backend, right)?  Is there a performance difference between casting those directly, taking index 0, vs casting to float at the end?

Reply:
yep, both `abs_normal_intensities` and `ldint` are now always arrays. I can't imagine there is any appreciable difference between those scenarios given a plethora of other bottlenecks. :)